csharp/tools/tracing-config.lua

```diff
@@ -174,7 +174,7 @@ function RegisterExtractorPack(id)
                     seenCompilerCall = true
                 end
                 if seenCompilerCall then
-                    table.insert(extractorArgs, '"' .. arg .. '"')
+                    table.insert(extractorArgs, QuoteArgument(arg))
```

Comment:
```suggestion
                    table.insert(extractorArgs, arg)
```

and change `commandLineString = table.concat(extractorArgs, " ")` below to  `commandLineString = ArgvToCommandLineString(extractorArgs)`

Reply:
I must be tired. Thank you for being thorough!